### PR TITLE
chore(eslint): update to latest standard

### PR DESCRIPTION
**Pull request by code-forger**
Update to the latest eslint standard

In general I preferred to keep odd things I saw rather than risk semantic changes.

#### Check boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [x] Dependency update
- [ ] Security update

#### Check boxes that apply:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

File: packages/one-app-bundler/webpack/loaders/meta-data-loader.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const { META_DATA_KEY } = require('../..');
 
 function metaDataLoader(content) {
   const { packageJson: { version } } = readPkgUp.sync();
-  const match = content.match(/export\s+default\s+(?!from)([\w\d]+);$/m);
+  const match = content.match(/export\s+default\s+(?!from)(\w+);$/m);
```
Comment (Matthew-Mallimo) [inline packages/one-app-bundler/webpack/loaders/meta-data-loader.js:20]: Why is the regex changing?

File: packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js [hunk 2]
```diff
@@ -18,12 +18,12 @@ const path = require('path');
...
 const cssBasePathString = JSON.stringify(path.resolve(__dirname, 'css-base.js'));
 
-const CSS_LOADER_FINDER = /var ___CSS_LOADER_API_IMPORT___ = (__webpack_){0,1}require(__){0,1}\([.a-zA-Z0-9/_*!\s-]*"[.a-zA-Z0-9/_]+\/css-loader\/dist\/runtime\/api.js"\);\n\s*exports = ___CSS_LOADER_API_IMPORT___\((undefined|false)?\);/;
+const CSS_LOADER_FINDER = /var ___CSS_LOADER_API_IMPORT___ = (__webpack_){0,1}require(__){0,1}\([\s\w!*./-]*"[\w./]+\/css-loader\/dist\/runtime\/api.js"\);\n\s*exports = ___CSS_LOADER_API_IMPORT___\((undefined|false)?\);/;
 
 // The following two regex patterns split the above regex in two
 // this is due to some UI libraries injecting their own vars between
 // var ___CSS_LOADER_API_IMPORT___ and exports = ___CS_LOADER_API_IMPORT___
-const CSS_RUNTIME_FINDER = /var ___CSS_LOADER_API_IMPORT___ = (__webpack_){0,1}require(__){0,1}\([.a-zA-Z0-9/_*!\s-]*"[.a-zA-Z0-9/_]+\/css-loader\/dist\/runtime\/api.js"\);/;
+const CSS_RUNTIME_FINDER = /var ___CSS_LOADER_API_IMPORT___ = (__webpack_){0,1}require(__){0,1}\([\s\w!*./-]*"[\w./]+\/css-loader\/dist\/runtime\/api.js"\);/;
```
Comment (Matthew-Mallimo) [inline packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js:26]: Why is the regex changing?

File: packages/one-app-bundler/webpack/loaders/meta-data-loader.js (hunk 1 above)
Comment (code-forger) [reply packages/one-app-bundler/webpack/loaders/meta-data-loader.js:20]: It's semantically identical and simpler

File: packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js (hunk 2 above)
Comment (code-forger) [reply packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js:26]: It's semantically identical and simpler

**Review by Matthew-Mallimo — DISMISSED** [review]

**Review by giulianok — DISMISSED** [review]

File: packages/holocron-dev-server/src/middleware/parrot-scenarios.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ export async function createHotParrotMiddleware({
         const mockFiles = watched[mockPath];
         return mockFiles.map((fileName) => path.join(mockPath, fileName));
       })
-      .reduce((array, next) => array.concat(next), []);
+      .reduce((array, next) => [...array, next], []);
```
Comment (JAdshead) [inline packages/holocron-dev-server/src/middleware/parrot-scenarios.js:93]: this is not the same. Array.prototype.concat allows for next to be an array or not and wont result in nested arrays. 
`[].concat('something') === [].concat(['something'])` 
`[...array, next] !== [...array, [next]]`

**Review by JAdshead — CHANGES_REQUESTED** [review]
revert concat changes

**Review by JAdshead — APPROVED** [review]

**Review by giulianok — APPROVED** [review]

**Pull request merged**